dips/dip-20.md

```diff
@@ -507,7 +441,7 @@ operations are as follows, and may be called by anyone:
 ## <a name="currency_ops">Currency Operations</a>
 
 There are a number of different operations that may be performed to view and update the
-metadata for a given currency (viz. [Required information for registration of a currency
+metadata for a given currency (viz., [required information for registration of a currency
 and metadata](#metadata_spec)). They can be divided into "getters" (non-mutative),
```

Comment:
I'm not sure what is meant by "...and metadata" here. When you register a currency, you specify the metadata, so it seems like there is only one thing, not "currency and metadata"